### PR TITLE
migrating to ossec-wazuh

### PR DESCRIPTION
**Pull request by akhon**
http://wazuh-documentation.readthedocs.org/en/latest/wazuh_installation.html

### CODE REVIEW

File: defaults/main.yml [hunk 1]
```diff
@@ -60,19 +70,23 @@ ossec_active_responses:
...
-    location: /var/log/xferlog
-  - log_format: syslog
-    location: /var/log/maillog
-  - log_format: apache
-    location: /var/www/logs/access_log
-  - log_format: apache
-    location: /var/www/logs/error_log
+    location: /var/log/dpkg.log
+
+ossec_local_commands:
```
Comment (mahmoudimus) [inline defaults/main.yml]: @akhon what does this do?

File: defaults/main.yml [hunk 2]
```diff
@@ -47,6 +53,10 @@ ossec_commands:
     executable: disable-account.sh
     expect: user
     timeout_allowed: yes
+  - name: route-null
```
Comment (mahmoudimus) [inline defaults/main.yml]: what does this do?

File: defaults/main.yml [hunk 3]
```diff
@@ -11,14 +11,24 @@ ossec_syscheck_directories:
...
+  - /etc/utmpx
+  - /etc/cups/certs
+  - /etc/dumpdates
+  - /etc/svc/volatile
 
 ossec_rootcheck_rootkit_files: /var/ossec/etc/shared/rootkit_files.txt
 ossec_rootcheck_rootkit_trojans: /var/ossec/etc/shared/rootkit_trojans.txt
+ossec_rootcheck_system_audit:
+  - /var/ossec/etc/shared/system_audit_rcl.txt
+  - /var/ossec/etc/shared/cis_debian_linux_rcl.txt
```
Comment (mahmoudimus) [inline defaults/main.yml]: Center for Information Security hardening

File: tasks/main.yml [hunk 4]
```diff
@@ -1,55 +1,61 @@
 ---
-- apt_key:
-    # If you need to change the key use at least gpg --keyid-format long keyfile
-    # the 32-bit short format is easily collided and insecure.
-    url=http://ossec.wazuh.com/repos/apt/conf/ossec-key.gpg.key
-    id=14B9C8DB9A1B1C65
+- name: clone ossec-wazuh git repo
+  shell: cd /opt && rm -rf ossec-wazuh && git clone https://github.com/wazuh/ossec-wazuh
```
Comment (mahmoudimus) [inline tasks/main.yml]: let's not do this. i do not want to install git on the ossec servers.

lets change this to use http://docs.ansible.com/ansible/file_module.html 

- `file: path=/opt/ossec-wazuh state=absent`

then  use the fetch file.

Comment (mahmoudimus) [inline tasks/main.yml]: ok, talked to @akhon he's saying that these are the supported installation procedures. he's right. however, we need to build our own debian packages, but this is OK for now.

http://wazuh-documentation.readthedocs.org/en/latest/wazuh_installation.html

Comment (mahmoudimus) [inline tasks/main.yml]: @akhon I think you should use the file module + the git module

**Comment by mahmoudimus** [comment]
@akhon ok, let me know when you've addressed these comments.

**Comment by akhon** [comment]
@mahmoudimus:
- used git and file ansible modules instead of commands
- about defaults: used defaults from default ossec-wazuh config file, `ossec_local_commands` are active response: http://ossec-docs.readthedocs.org/en/latest/syntax/head_ossec_config.active-response.html

**Comment by akhon** [comment]
let me know if all commits in PR needs to be rebased into one commit

File: tasks/main.yml [hunk 5]
```diff
@@ -1,55 +1,68 @@
...
+    path=/opt/ossec-wazuh
+    state=absent
 
-- apt_repository:
-    repo='deb http://ossec.wazuh.com/repos/apt/ubuntu/ {{ansible_distribution_release}} main'
-    state=present
+- name: clone git repo
+  git:
+    repo=https://github.com/wazuh/ossec-wazuh
+    dest=/opt/ossec-wazuh
```
Comment (mahmoudimus) [inline tasks/main.yml]: Can we pin this to a version?

Comment (akhon) [inline tasks/main.yml]: sure!

Comment (mahmoudimus) [inline tasks/main.yml]: Also it might be helpful for depth=1

File: tasks/main.yml [hunk 6]
```diff
@@ -1,55 +1,68 @@
...
+  file:
+    path=/opt/ossec-wazuh
+    state=absent
 
-- apt_repository:
-    repo='deb http://ossec.wazuh.com/repos/apt/ubuntu/ {{ansible_distribution_release}} main'
-    state=present
+- name: clone git repo
+  git:
+    repo=https://github.com/wazuh/ossec-wazuh
```
Comment (mahmoudimus) [inline tasks/main.yml]: the repo here should end in .git

Comment (akhon) [inline tasks/main.yml]: this is not required in https

File: tasks/main.yml (hunk 5 above)
Comment (akhon) [inline tasks/main.yml]: done!

**Pull request merged**